.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install Ruby (2.7)
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7.2
+        
+    - name: Install NodeJS (16)
+      uses: actions/setup-node@v2
+      with:
+        node-version: '16'
+
+    - name: Set up Code Climate test-reporter
+      run: |
+        curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+        chmod +x ./cc-test-reporter
+        ./cc-test-reporter before-build
+    
+    - name: Install bundler
+      run: |
+        gem install bundler
+
+    - name: Install dependencies
+      run: |
+        bundle install
+    
+    - name: Install node_modules
+      run: |
+        npm install
+    
+    - name: Set up database
+      run: |
+        bundle exec rake db:create db:migrate
+    
+    - name: run RSpec tests and capture coverage
+      run: |
+        bundle exec rspec
+        ./cc-test-reporter format-coverage --output coverage/codeclimate.$SUITE.json --input-type simplecov
+    
+    - name: run Cucumber tests and capture coverage
+      run: |
+        bundle exec cucumber --retry 2
+        ./cc-test-reporter format-coverage --output coverage/codeclimate.$SUITE.json --input-type simplecov
+        
+    - name: run ESLinting
+      run: |
+        npm run lint
```

Comment:
I notice there are 12 lint warnings, are you okay with all of those? Typically if I see a lint warning I'll either address the issue or add an inline comment to tell ESlint I'm okay with it. (So that future developers dont have to wonder).

Warning:    49:26  warning  'map' is defined but never used                                      no-unused-vars
Warning:   100:22  warning  'markerOnClick' was used before it was defined                       no-use-before-define
Warning:   [11](https://github.com/debbieyuen/baytrailbirding/runs/5875794498?check_suite_focus=true#step:12:11)9:22  warning  'locMarkerOnClick' was used before it was defined                    no-use-before-define
Warning:   [12](https://github.com/debbieyuen/baytrailbirding/runs/5875794498?check_suite_focus=true#step:12:12)4:71  warning  'parallelHitEbirdEndpoint' was used before it was defined            no-use-before-define
Warning:   125:8   warning  'parallelHitEbirdEndpoint' was used before it was defined            no-use-before-define
Warning:   128:25  warning  'e' is defined but never used                                        no-unused-vars
Warning:   129:[13](https://github.com/debbieyuen/baytrailbirding/runs/5875794498?check_suite_focus=true#step:12:13)  warning  Unexpected 'this'                                                    no-invalid-this
Warning:   133:11  warning  Identifier name 'locMarkerOnClick' is too long (> [15](https://github.com/debbieyuen/baytrailbirding/runs/5875794498?check_suite_focus=true#step:12:15))                id-length
Warning:   [17](https://github.com/debbieyuen/baytrailbirding/runs/5875794498?check_suite_focus=true#step:12:17)5:11  warning  'createPos' is defined but never used                                no-unused-vars
Warning:   [18](https://github.com/debbieyuen/baytrailbirding/runs/5875794498?check_suite_focus=true#step:12:18)2:3   warning  'parallelHitEbirdEndpoint' was used before it was defined            no-use-before-define
Warning:   186:2   warning  Async function 'parallelHitEbirdEndpoint' has no 'await' expression  require-await
Warning:   186:17  warning  Identifier name 'parallelHitEbirdEndpoint' is too long (> 15)        id-length
Warning:   [20](https://github.com/debbieyuen/baytrailbirding/runs/5875794498?check_suite_focus=true#step:12:20)3:3   warning  'hitEbirdEndpoint' was used before it was defined                    no-use-before-define
Warning:   205:11  warning  Identifier name 'hitEbirdEndpoint' is too long (> 15)                id-length
Warning:   [23](https://github.com/debbieyuen/baytrailbirding/runs/5875794498?check_suite_focus=true#step:12:23)3:12  warning  Unexpected 'this'                                                    no-invalid-this